lxd/network/driver_ovn.go

```diff
@@ -3218,6 +3282,12 @@ func (n *ovn) Update(newNetwork api.NetworkPut, targetNode string, clientType re
 		delete(newNetwork.Config, ovnVolatileUplinkIPv6)
 	}
 
+	// Update port configurations for VLAN if necessary.
+	err = n.configurePortsForVLAN(newNetwork.Config["vlan"], true)
```

Comment:
where do changes to uplink ports get applied during updates currently? Can we move this call there?